tlsproxy.go

```diff
@@ -7,14 +7,16 @@ import (
 	_ "net/http/pprof"
 	"time"
 
+	"github.com/getlantern/golog"
 	"github.com/getlantern/http-proxy/buffers"
 	"github.com/getlantern/netx"
-	"github.com/siddontang/go/log"
```

Comment:
Don't believe we wanted to be using the above logger, but perhaps that was intentional?